django/db/backends/mysql/base.py

```diff
@@ -270,6 +295,9 @@ def init_connection_state(self):
                 # for NULL. Disabling this brings this aspect of MySQL in line
                 # with SQL standards.
                 cursor.execute('SET SQL_AUTO_IS_NULL = 0')
+        if self.tx_isolation_level != 'repeatable read':  # change from MySQL default
+            with self.cursor() as cursor:
+                cursor.execute('SET SESSION TRANSACTION ISOLATION LEVEL %s' % self.tx_isolation_level)
```

Comment:
it can't really be one statement, because `SET TRANSACTION ISOLATION LEVEL` is a different statement from `SET var = value` as far as I understand.

Does concatenating statements with semicolons work? 

I thought about `init_command`, but yes, the user already has control over that, and I don't want to start guessing what they did there.